dashboard/app/lib/javascripts/dashboard/views/system-applications-link.js.jsx

```diff
@@ -0,0 +1,26 @@
+import { extend } from 'marbles/utils';
+import { pathWithParams } from 'marbles/history';
+
+var SystemApplicationsLink = React.createClass({
+	displayName: "Views.SystemApplicationLink",
+
+	render: function () {
+		var props = extend({}, this.props);
+		props.href = this.__getHref();
+		props.className = "system-applications-link";
+		var label = (this.props.showSystemApps ? "Hide" : "Show") +
+			" System Applications";
+		return React.createElement('a', props, label);
```

Comment:
JSX is preferred over using the React API directly. This could be simplified to:

```jsx
return (
  <a href={this.__getHref()} className="system-applications-link">
    {this.props.showSystemApps ? 'Hide' : 'Show'} System Applications
  </a>
);
```